gdk4-wayland/src/wayland_device.rs

```diff
@@ -3,12 +3,17 @@
 use crate::WaylandDevice;
 #[cfg(any(feature = "wayland_crate", feature = "dox"))]
 #[cfg_attr(feature = "dox", doc(cfg(feature = "wayland_crate")))]
-use glib::translate::ToGlibPtr;
+use crate::{glib::translate::ToGlibPtr, gdk::prelude::*};
 #[cfg(any(feature = "wayland_crate", feature = "dox"))]
 #[cfg_attr(feature = "dox", doc(cfg(feature = "wayland_crate")))]
 use wayland_client::{
-    protocol::{wl_keyboard::WlKeyboard, wl_pointer::WlPointer, wl_seat::WlSeat},
-    sys::client::wl_proxy,
+    backend::ObjectId,
+    protocol::{
+        __interfaces::{WL_KEYBOARD_INTERFACE, WL_POINTER_INTERFACE, WL_SEAT_INTERFACE},
```

Comment:
Could use the implementations of `Proxy` with the `interface` function.

Since you already import `Proxy`, you'd use `WlSeat::interface()` for example

Reply:
Thanks